dao/src/main/java/org/thingsboard/server/dao/sql/edge/JpaBaseEdgeEventDao.java

```diff
@@ -189,20 +198,36 @@ public PageData<EdgeEvent> findEdgeEvents(UUID tenantId, EdgeId edgeId, TimePage
 
     @Override
     public void cleanupEvents(long ttl) {
-        log.info("Going to cleanup old edge events using ttl: {}s", ttl);
-        try (Connection connection = dataSource.getConnection();
-             PreparedStatement stmt = connection.prepareStatement("call cleanup_edge_events_by_ttl(?,?)")) {
-            stmt.setLong(1, ttl);
-            stmt.setLong(2, 0);
-            stmt.setQueryTimeout((int) TimeUnit.HOURS.toSeconds(1));
-            stmt.execute();
-            printWarnings(stmt);
-            try (ResultSet resultSet = stmt.getResultSet()) {
-                resultSet.next();
-                log.info("Total edge events removed by TTL: [{}]", resultSet.getLong(1));
-            }
-        } catch (SQLException e) {
-            log.error("SQLException occurred during edge events TTL task execution ", e);
+        partitioningRepository.dropPartitionsBefore(TABLE_NAME, ttl, TimeUnit.HOURS.toMillis(partitionSizeInHours));
+    }
+
+    @Override
+    public void migrateEdgeEvents() {
+        long startTime = edge_events_ttl > 0 ? System.currentTimeMillis() - TimeUnit.SECONDS.toMillis(edge_events_ttl) : 1480982400000L;
```

Comment:
please replace the default date:
Tue Dec 06 **2020** 00:00:00 instead of Tue Dec 06 **2016** 00:00:00

Reply:
Actually, let's set it to **Aug 17, 2021** - that's the release date of Edge product